db/seeds.rb

```diff
@@ -13,38 +13,41 @@
                                 { title: 'DevOps' }
                               ])
 
+users = User.create!([
+                         { email: 'admin@testguru.com', name: 'Admin', password: 'supersecretpassword1337', score: 1000 },
+                         { email: 'petya@bulochkin.ru', name: 'Petya Bulochkin', password: 'callmepetya333', score: 0 }
+                     ])
+
 tests = Test.create!([
-                       { title: 'Ruby', level: 1, category_id: categories[0].id },
-                       { title: 'RubyOnRails', level: 2, category_id: categories[0].id },
-                       { title: 'Javascript', level: 1, category_id: categories[1].id },
-                       { title: 'SQL', level: 1, category_id: categories[2].id },
-                       { title: 'Docker', level: 3, category_id: categories[3].id }
+                       { title: 'Ruby', level: 1, category: categories[0], author_id: users[0].id },
```

Comment:
```suggestion
                       { title: 'Ruby', level: 1, category: categories[0], author: users[0] },
```
для авторства тоже можно использовать ассоциацию (если правильно написать миграции и ассоциации)